tests/integration/test_npm.py

```diff
@@ -0,0 +1,66 @@
+import logging
+from pathlib import Path
+
+import pytest
+
+from . import utils
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize(
+    "test_params,lockfile_version",
+    [
+        # Test case checks loading npm dependencies in SBOM format for source repo
+        # Plus fetching the dependencies
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/npm-cachi2-smoketest.git",
+                ref="cb56134a2543ba56c303b4cd8e7c174cef9de4ea",
+                packages=({"path": ".", "type": "npm"},),
+            ),
+            1,
+            id="npm_smoketest_lockfile1",
+        ),
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/npm-cachi2-smoketest.git",
+                ref="05f841a5b94fb447b4ed428a37d7395d66d9fc6a",
+                packages=({"path": ".", "type": "npm"},),
+            ),
+            2,
+            id="npm_smoketest_lockfile2",
+        ),
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/npm-cachi2-smoketest.git",
+                ref="4baf3d58db432752aa63156597b28a7e775fd862",
+                packages=({"path": ".", "type": "npm"},),
+            ),
+            3,
+            id="npm_smoketest_lockfile3",
+        ),
+    ],
+)
+def test_npm_smoketest(
+    test_params: utils.TestParameters,
+    lockfile_version: int,
+    cachi2_image: utils.ContainerImage,
+    tmp_path: Path,
+    test_data_dir: Path,
+    request: pytest.FixtureRequest,
+):
+    """
+    Smoketest for npm offline install development.
+
+    :param test_params: Test case arguments
+    :param lockfile_version: Integer version to specify semantics for package-lock.json
```

Comment:
will lockfile_version be used for anything?

Reply:
This should probably be read from the cloned lockfile.

Reply:
OK removed